tables/README.md

```diff
@@ -12,13 +12,9 @@ Table S2: DNA results table
 - This excel file aims to include key results from analyzing DNA sequencing results 
     - Sheet 1: this table shows tumor mutation burden (TMB) for all samples - `TMB_coding` is calculated based on mutations in coding regions and `TMB_all` is calculated by all mutations a sample has
         - Input files `pbta-snv-mutation-tmb-all.tsv` and `pbta-snv-mutation-tmb-coding.tsv` are in data release.
-    - Sheet 2: this table shows signature weights for all [COSMIC signatures](https://cancer.sanger.ac.uk/cosmic) per sample
-        - Input file `cosmic_signatures_results.tsv` is generated from module [mutational-signatures](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/mutational-signatures)
-    - Sheet 3: this table shows signature weights for all [Alexandrov et al., 2013](https://www.ncbi.nlm.nih.gov/pubmed/23945592) signatures per sample
-        - Input file `nature_signatures_results.tsv` is generated from module [mutational-signatures](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/mutational-signatures)
-    - Sheet 4: this table shows signature weights for CNS signature identified in [Degasperi et al, 2020](https://doi.org/10.1038/s43018-020-0027-5) using [`sigfit`](https://github.com/kgori/sigfit) per sample
-        - Input file fitted_cns_signature_exposures.RDS` is generated from module [mutational-signatures](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/mutational-signatures)
-    - Sheet 5: this table summarizes the number of chromothripsis events per sample 
+    - Sheet 2: this table shows signature weights for CNS signature identified in [Degasperi et al, 2020](https://doi.org/10.1038/s43018-020-0027-5) using [`sigfit`](https://github.com/kgori/sigfit) per sample
+        - Input file `fitted_exposures_signal-cns_sigfit.rds` is generated from module [mutational-signatures](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/mutational-signatures)
+    - Sheet 3: this table summarizes the number of chromothripsis events per sample 
```

Comment:
```suggestion
    - Sheet 3: this table summarizes the number of chromothripsis events per sample 
```

Just one more request here to add the input file since the other tabs have this info

Reply:
I thought it is on line 18? 
```
Input file `chromothripsis_summary_per_sample.txt` is generated from module [chromothripsis]`
```

Reply:
You're right! I got tricked by hidden lines.